express/blocks/plans-comparison/plans-comparison.js

```diff
@@ -0,0 +1,397 @@
+  const $featuresWrapper = createTag('ul', { class: 'features-wrapper' });
+  value.features.forEach((feature) => {
+    $featuresWrapper.append(feature);
+  });
+
+  return $featuresWrapper;
+}
+
+function decorateCTAs($block, payload, value) {
+  const $buttonsWrapper = createTag('ul', { class: 'ctas-wrapper' });
+  value.ctas.forEach((cta, index) => {
+    $buttonsWrapper.append(cta);
+    if (index === 0) {
+      cta.classList.add('primary');
+    }
+
+    cta.addEventListener('click', (e) => {
+      e.stopPropagation();
+    });
+  });
+
+  return $buttonsWrapper;
+}
+
+function decorateCards($block, payload) {
+  const $cardsWrapper = createTag('div', { class: 'plans-comparison-cards' });
+
+  $block.append($cardsWrapper);
+
+  for (const [key, value] of Object.entries(payload)) {
+    if (['free', 'premium'].includes(key)) {
+      const $card = createTag('div', { class: `plans-comparison-card card-${key}` });
+      const $contentTop = createTag('div', { class: 'content-top' });
+      const $contentBottom = createTag('div', { class: 'content-bottom' });
+      const $heading = createTag('div', { class: 'plans-comparison-heading' });
+      const $subCopy = createTag('p', { class: 'plans-comparison-sub-copy' });
+      const $features = decorateFeatures($block, payload, value);
+      const $ctas = decorateCTAs($block, payload, value);
+
+      $cardsWrapper.append($card);
+      $contentTop.append($heading, $subCopy, $features);
+      $contentBottom.append($ctas);
+      $heading.innerHTML = value.heading;
+      $subCopy.textContent = value.subCopy;
+      $card.append($contentTop, $contentBottom);
+
+      decorateToggleButton($block, $card, payload);
+
+      if (key === 'premium') {
+        expandCard($card, payload);
+        payload.cardHeight = $card.offsetHeight;
+        collapseCard($card, payload);
+      }
+
+      $card.classList.add('transition');
+    }
+  }
+}
+
+function decoratePagination($block, payload) {
+  const $paginationWrapper = createTag('div', { class: 'pagination-wrapper' });
+  const $cards = $block.querySelectorAll('.plans-comparison-card');
+  if ($cards) {
+    Array.from($cards)
+      .forEach(($card) => {
+        const $paginationPill = createTag('span', { class: 'pagination-pill' });
+        $paginationWrapper.append($paginationPill);
+
+        $paginationPill.addEventListener('click', () => {
+          toggleExpandableCard($block, $card, payload);
+        });
+      });
+    $block.append($paginationWrapper);
+  }
+}
+
+export default function decorate($block) {
+  let payload;
+  const locale = getLocale(new URL(window.location));
+  const $linkList = document.querySelector('.link-list-container');
+  let fragmentUrl;
+  if (locale === 'us') {
+    fragmentUrl = 'https://main--express-website--adobe.hlx.page/express/fragments/plans-comparison';
+  } else {
+    fragmentUrl = `https://main--express-website--adobe.hlx.page/${locale}/express/fragments/plans-comparison`;
```

Comment:
You should remove the host here, especially for production, it should not go to hlx.page...